docs/ai-coder/custom-agents.md

```diff
@@ -31,19 +21,18 @@ From there, the agent can run the MCP server with the `coder exp mcp server` com
 Inside a Coder workspace, run the following commands:
 
 ```sh
-coder login # be sure to be authenticated with the Coder CLI
-export CODER_MCP_APP_STATUS_SLUG=my-agent # needs to be the same as the slug in the coder_app resource
+coder login
+export CODER_MCP_APP_STATUS_SLUG=my-agent
 
 # Use your own agent's logic and syntax here:
 any-custom-agent configure-mcp --name "coder" --command "coder exp mcp server"
 ```
 
 This will start the MCP server and report activity back to the Coder control plane on behalf of the coder_app resource.
 
+> [!NOTE]
 > See the [Goose module](https://github.com/coder/registry/blob/main/registry/coder/modules/goose/main.tf) source code for a real world example.
```

Comment:
The module on main will be updated with agentapi soon and won't be a great example for how to add a custom agent. Let's reference a specific tag instead.

```suggestion
> See the [Goose module](https://github.com/coder/registry/blob/release/coder/goose/v1.3.0/registry/coder/modules/goose/main.tf) source code for a real world example.
```